assets/global.js

```diff
@@ -505,7 +506,7 @@ class SliderComponent extends HTMLElement {
     this.sliderItems = this.querySelectorAll('[id^="Slide-"]');
     this.enableSliderLooping = false;
     this.currentPageElement = this.querySelector('.slider-counter--current');
-    this.pageTotal = this.querySelector('.slider-counter--total');
+    this.pageTotalElement = this.querySelector('.slider-counter--total');
```

Comment:
I wonder if pageTotalElement is the best. Since for some uses cases it's not the total amount of elements but pages. As a page/view can have 1-3 slides within it: 
![](https://screenshot.click/21-12-202im-63hg8.png)

Reply:
I added the `Element` to the variable to make it clear that it's a NodeElement. Isn't that always the case? Would you rather use `Node`?

`this.pageTotal` sounded like an integer variable to me :P